grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -150,6 +152,12 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
         }?:0
     }
 
+    Integer getQuantityInShipmentsInStandardUom() {
```

Comment:
This is already added in the combined shipments feature

Reply:
But this feature branch is not yet merged to develop.